src/main/java/uk/gov/companieshouse/insolvency/data/service/InsolvencyServiceImpl.java

```diff
@@ -63,10 +64,11 @@ public void processInsolvency(String contextId, String companyNumber,
                 InsolvencyDocument insolvencyDocumentFromDb =
                         insolvencyDocumentFromDbOptional.get();
 
-                String updatedAtFromDbStr = insolvencyDocumentFromDb
-                        .getUpdated().getAt();
+                LocalDateTime updatedAtFromDbLocalDateTime = insolvencyDocumentFromDb
+                        .getUpdatedAt();
 
-                OffsetDateTime updatedAtFromDb = OffsetDateTime.parse(updatedAtFromDbStr);
+                OffsetDateTime updatedAtFromDb =
+                        OffsetDateTime.of(updatedAtFromDbLocalDateTime, ZoneOffset.UTC);
 
                 if (dateFromBodyRequest.isAfter(updatedAtFromDb)) {
```

Comment:
Please compare the new delta at coming from the body request with the delta at stored in the db and not the updated at as currently implemented. Thanks.